app/assets/scss/components/_attachment.scss

```diff
@@ -0,0 +1,24 @@
+.attachment {
+  @include tablet {
+    align-items: center;
+    display: flex;
+    justify-content: space-between;
+  }
+  & > a {
+    &:after {
+      color: #555555;
+      content: attr(data-details);
```

Comment:
Wywalą błąd walidatory, bo nie wszystkie czytniki czytają content. Możemy to zrobić inaczej?